CI failing with mounting volume error

https://github.com/ZcashFoundation/zebra/actions/runs/6382508836/job/17321768540?pr=7653#step:13:188

```
docker: Error response from daemon: error while mounting volume '/var/lib/docker/volumes/fully-synced-rpc-85b855b/_data': failed to mount local volume: mount /dev/sdb:/var/lib/docker/volumes/fully-synced-rpc-85b855b/_data: device or resource busy.
Error: Process completed with exit code 125.
```

This seems to be happening for every PR we have opened *recently*. We can post more links to failures and more information in this ticket.

--- a/.github/workflows/deploy-gcp-tests.yml
+++ b/.github/workflows/deploy-gcp-tests.yml
@@ -432,6 +432,8 @@ jobs:
           --ssh-flag="-o ConnectTimeout=5" \
           --command \
           "\
+          # Wait for the disk to be attached
+          while [[ ! -e /dev/sdb ]]; do sleep 1; done && \
           sudo docker run \
           --name ${{ inputs.test_id }} \
           --tty \
@@ -480,6 +482,8 @@ jobs:
           --ssh-flag="-o ConnectTimeout=5" \
           --command \
           "\
+          # Wait for the disk to be attached
+          while [[ ! -e /dev/sdb ]]; do sleep 1; done && \
           sudo docker run \
           --name ${{ inputs.test_id }} \
           --tty \

